### PR TITLE
MNT Use shared travis config, use sminnee/phpunit

### PR DESCRIPTION
**Pull request by emteknetnz**
Merge up to 2.1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,4 @@
...
-    - php: 7.2
-      env: DB=MYSQL PHPUNIT_TEST=1
-
-before_script:
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-
-  - composer validate
-  - if [[ $DB == "PGSQL" ]]; then composer require --no-update silverstripe/postgresql 2.0.x-dev; fi
-  - composer require --no-update silverstripe/recipe-cms 1.0.x-dev
```
Comment (dnsl48) [inline .travis.yml:24]: looks like it only tests against the old version...

**Review by dnsl48 — CHANGES_REQUESTED** [review]
The module doesn't seem compatible with the "standard" matrix

**Comment by emteknetnz** [comment]
OK I'm thinking we just leave this one red.  There's no benefit in testing this against such an old recipe just to get this build green as this gives a false sense of module health

Other than the old recipe version, there's nothing special about this module so my recommendation is we just merge this as is and work out if/when we fix this module up later on

We won't be releasing a new version of this for the 4.7 release

**Comment by dnsl48** [comment]
Fair enough, true positive is better than false negative.

**Review by dnsl48 — APPROVED** [review]
the broken tests are legit

**Pull request merged**